01_fib/harirakr/main_test.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNegativeFib(t *testing.T) {
+	assert.Equal(t, []int64{1, -1, 2, -3, 5, -8, 13, -21, 34, -55}, fib(-10))
+	assert.Equal(t, []int64{1, -1, 2}, fib(-3))
+	assert.Equal(t, []int64{1, -1}, fib(-2))
+	assert.Equal(t, []int64{1}, fib(-1))
+	assert.Equal(t, []int64{}, fib(0))
+}
+
+func TestFib(t *testing.T) {
+	assert.Equal(t,
+		[]int64{1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987, 1597, 2584, 4181, 6765},
+		fib(20))
+	assert.Equal(t, []int64{1, 1, 2, 3, 5, 8, 13, 21, 34, 55}, fib(10))
+	assert.Equal(t, []int64{1, 1}, fib(2))
+	assert.Equal(t, []int64{1}, fib(1))
+	assert.Equal(t, []int64{}, fib(0))
+}
+
+func TestMain(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := strconv.Quote("[1 1 2 3 5 8 13]\n")
+	actual := strconv.Quote(buf.String())
+	if expected != actual {
```

Comment:
you're using `assert.Equal` above. Why not use it also here? I believe it's a better fit.

Reply:
Don.